AccountsManager/src/main/java/di/AccountsManagerModule.java

```diff
@@ -0,0 +1,39 @@
+package di;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.AbstractModule;
+import configuration.MybatisConfiguration;
+import io.logz.guice.jersey.JerseyModule;
+import io.logz.guice.jersey.configuration.JerseyConfiguration;
+import org.apache.ibatis.reflection.factory.DefaultObjectFactory;
+import org.apache.ibatis.reflection.wrapper.DefaultObjectWrapperFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+public class AccountsManagerModule extends AbstractModule {
+
+    public AccountsManagerModule(){
+
+    }
+
+    @Override
+    protected void configure() {
+        ObjectMapper mapper = new ObjectMapper();
+        binder().requireExplicitBindings();
```

Comment:
Let's find a proper way to ensure require explicit bindings in every services (i.e. have a method `static Inject createInjectorWithExplicitBindings(Module... modules)` that all your `Main` classes will call, instead of `Guice.createInjector`)